pkg/webhook/webhook.go

```diff
@@ -490,3 +511,49 @@ func (wh *webhook) validateCVC(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionRe
 		"configured for operations other than UPDATE")
 	return response
 }
+
+// verifyExistenceOfSnapshots returns error when there are volumesnapshots or volumesnapshotDatas
+// related to volume and if any error occured else return nil
+func (wh *webhook) verifyExistenceOfSnapshots(pvc *corev1.PersistentVolumeClaim) error {
+	volumeSnapshotList, err := wh.snapClientSet.
+		VolumesnapshotV1().
+		VolumeSnapshots(pvc.Namespace).
+		List(metav1.ListOptions{
+			LabelSelector: snapshotMetadataPVName + "=" + pvc.Spec.VolumeName,
+		})
+	if err != nil {
+		return errors.Wrapf(err, "failed to list snapshots related to volume: %s", pvc.Spec.VolumeName)
+	}
+	if len(volumeSnapshotList.Items) > 0 {
+		return errors.Errorf("pvc %s has '%d' snapshot(s)", pvc.Name, len(volumeSnapshotList.Items))
```

Comment:
nitl: ( "pvc %s  has '%d' number of snapshot(s) exists", pvc.Name, len(volumeSnapshotList.Items)) ?

Reply:
How about adding the word
`( "pvc %s has '%d' number of depndent snapshot(s) exists", pvc.Name, len(volumeSnapshotList.Items))`